File: CV.UI/src/app/models/work-experience.ts
export class WorkExperience {
    constructor(name:string, startDate: Date, endDate: Date, type: string, description: string) {
        this.name = name;
        this.startDate = startDate;
        this.endDate = endDate;
        this.type = type;
        this.description = description;
    }
    name: string;
    startDate: Date;
    endDate: Date;
    type: string;
    description: string;
}

File: CV.API/CV.API/Domain/Models/WorkExperience.cs
namespace CV.API.Domain.Models;
public class WorkExperience : IEntity
{
    public WorkExperience(string name, DateTime startDate, DateTime endDate, string type, string description)
    {
        Name = name;
        StartDate = startDate;
        EndDate = endDate;
        Type = type;
        Description = description;
    }
    public int Id { get; set; }
    public string Name { get; set; }
    public DateTime StartDate { get; set; }
    public DateTime EndDate { get; set; }
    public string Type { get; set; }
    public string Description { get; set; }
}